### ISSUE TITLE
Can BLUES perform multiple ligands rotate and move

### ISSUE THREAD

**Issue by VincenzoChen** [MobleyLab/blues#172]
Hello, I have a system contains three ligands and I wanto to rotate and move all of them. When I use BLUES to perform such action, it raised an error "Particle coordinate is nan". So I wander whether BLUES can perform rotation and move on multiple ligands? Thanks!

**Comment by davidlmobley** [comment]
No, it can't. We were waiting on support for multiple alchemical regions from openmmtools; I believe David Huggins has now implemented this but we haven't been able to update BLUES to use it yet. ETA is "not anytime soon" because I don't have a student funded for further development of BLUES at present.

**Comment by VincenzoChen** [comment]
Thanks! I think I may try to add this function if I can .But I don't know when exactly.
